Marksman/Cst.fs

```diff
@@ -160,6 +163,7 @@ let rec private fmtElement =
     | WL x -> fmtWikiLink x
     | ML l -> fmtMdLink l
     | MLD r -> fmtMdLinkDef r
+    | YML y -> failwith "Format for yaml is not implemented"
```

Comment:
Do you plan to implement it? Just dumping the contents with range info (a-la TextNode) should be fine.

Reply:
still relevant